tests/test.sh

```diff
@@ -31,12 +31,28 @@ test_bin() {
   local expect=$1; shift
   local RESULT=-1
   cd $name;
-  bundle exec metadata-json-lint $* metadata.json >/dev/null 2>&1
+  bundle exec metadata-json-lint $* metadata.json >last_output 2>&1
   RESULT=$?
   if [ $RESULT -ne $expect ]; then
-    fail "Failing Test '${name}' (bin)"
+    fail "Failing Test (unexpected exit code) '${name}' (bin)"
   else
-    echo "Successful Test '${name}' (bin)"
+    # If the test is not expected to succeed then it should match an expected output
+    if [ $expect -eq $SUCCESS ]; then
```

Comment:
I would prefer not to; as some tests expect different results from rake than from metadata-json-lint directly; so this would require adding a bit more complexity to the test script.